src/ProjectTemplates/Web.ProjectTemplates/GrpcService-CSharp.csproj.in

```diff
@@ -4,6 +4,11 @@
     <TargetFramework>${DefaultNetCoreTargetFramework}</TargetFramework>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
+    <!--#if (NativeAot) -->
+    <PublishAot>true</PublishAot>
+    <TrimMode>full</TrimMode>
+    <PublishIISAssets>false</PublishIISAssets>
+    <!--#endif -->
```

Comment:
I copied this from this api template. Can `TrimMode` and `PublishIISAssets` be inferred from `PublishAot`?

Issue to clean up this change in the template: https://github.com/dotnet/aspnetcore/issues/46465

Reply:
This has already been cleaned up in the api template as the SDK defaults these now: https://github.com/dotnet/aspnetcore/blob/main/src/ProjectTemplates/Web.ProjectTemplates/Api-CSharp.csproj.in#L10-L12